### PR TITLE
Add First Time Setup message to the Help menu

### LINKED ISSUE #98: "First Time?" menu item
I need to write a "First Time?" menu button for the menu bar. 

The first item should be "First Time Setup" and should include a quick setup on how the program is setup and used. 

Under this should be the name of all our contests in order of the contest dropdown, and if clicked, provide help with each contest and should include blurbs on the contests and exchanges with info on the real dates and links to these contests.

I've assigned myself the texts to write and once done, we can program the menu buttons to be populated.

### PR DESCRIPTION
**Pull request by w7sst**
@scotthibbs suggested we add a simple `First Time Setup` menu pick to the Help menu. I used his suggested text as a starting point. Thanks to Scott for providing the original text. Fixes #98 

When you review this, please suggest changes to wording either as a comment, or make the changes directly via github.

See output below...
image

image

### CODE REVIEW

File: Main.pas [hunk 1]
```diff
@@ -1376,6 +1378,36 @@ procedure TMainForm.WipeBoxes;
...
+        '3) In the station section replace VE3NEA with your call.'#13 +
+        '4) Select your CW Speed, Tone, and Bandwidth.'#13 +
+        '5) Set Min/Max CW Receive speed using Settings menu.'#13 +
+        '6) Under band conditions you can select any realistic hardships.'#13 +
+        '7) Activity is the average amount of responses you want per CQ.'#13 +
+        '    So if no one responds, you might get twice the number the'#13 +
+        '    following time. This is a pile up trainer after all - have fun.'#13 +
+        '8) Select the time limit you want to run the contest.'#13 +
+        '9) The Run button has a drop down.'#13 +
+        '    - Single Call - immediately you will get someone trying to'#13 +
```
Comment (K6OK) [inline Main.pas:1397]: Minor comment: reverse order to match the pulldown list.  Minor wording suggestions.
```suggestion
        '    - Pile up - Hit Enter or F1 to call CQ to start. Get ready for pileups!'#13 
        '    - Single Calls - Work one station at a time with no pileups.'#13 +
```

File: Main.pas [hunk 2]
```diff
@@ -1376,6 +1378,37 @@ procedure TMainForm.WipeBoxes;
...
+        '6) Under band conditions you can select any realistic hardships.'#13 +
+        '7) Activity is the average amount of responses you want per CQ.'#13 +
+        '    So if no one responds, you might get twice the number the'#13 +
+        '    following time. This is a pile up trainer after all - have fun.'#13 +
+        '8) Select the time limit you want to run the contest.'#13 +
+        '9) The Run button has a drop down.'#13 +
+        '    - Pile up - Hit Enter or F1 to call CQ to start. Get ready for pileups!'#13 
+        '    - Single Calls - Work one station at a time with no pileups.'#13 +
+        '      contact you.'#13 +
+        '    - Pile up - You need to call CQ to start. hit enter or F1.'#13 +
```
Comment (scotthibbs) [inline Main.pas:1400]: The "contact you" seems to be dangling.. And pile up was inserted above. So line 1399 and 1400 need to be deleted right?

I know we discussed the run button. Might already been an issue (I think it is) but we mentioned having to change this run button (to say stop when started, move dropdown to near contest selection, lists contest that are selected above, etc)

**Review by scotthibbs — CHANGES_REQUESTED** [review]
Might not have done this right cuz I think I started another review. But I think several lines are hanging or duplicated. (1399 and 1400) Could someone check for me?

File: Main.pas (hunk 2 above)
Comment (w7sst) [reply Main.pas]: Good find! All repaired and ready to go. Thank you @K6OK for the wording changes.

Here is the updated output:
image

**Review by scotthibbs — APPROVED** [review]

**Pull request merged**